src/app/components/challenge/challenge.component.html

```diff
@@ -1,7 +1,7 @@
 <app-side-bar *ngIf="authService.isAuth"></app-side-bar>
 <app-header-static></app-header-static>
-<div class="web-container">
-    <div class="challenge-flex" [class.center]="!authService.isAuth">
+<div class="web-container" [style.center]="authService.isAuth">
+    <div class="challenge-flex">
```

Comment:
Why have you removed this from here now?

Reply:
Sorry about that. I am a little helter-skelter...Reverted!

Reply:
You need to add that this thing with `challenge-flex` in order to work.

Note: Please always check if its working on your local before pushing.

Reply:
I reverted the `!` too

Reply:
It will not work, you need to add that with the `challenge-flex` div.

> Note: Please always check if its working on your local before pushing.
. :smile: